manifests/templates/imageio.yaml.in

```diff
@@ -0,0 +1,50 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: imageio-deployment
+  namespace: {{ .Namespace }}
+  labels:
+    cdi.kubevirt.io/testing: ""
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: imageio
+  template:
+    metadata:
+      labels:
+        app: imageio
+        cdi.kubevirt.io/testing: ""
+    spec:
+      serviceAccountName: cdi-sa
+      containers:
+      - name: imageiotest
+        image: machacekondra/imageiotest
```

Comment:
@awels should those containers be part of CDI repo or is it ok to be external?

Reply:
Its okay to be external as long as it is some spot that can be accessed by the CI system. We use other external containers in other places already, like nginx and registry containers.

Reply:
If this is a cdi-specific thing (nginix is not) then we should have the wherewithal to produce the image as part of the build (as we do for the test http server).